CRM/Report/Form.php

```diff
@@ -4247,6 +4247,42 @@ public function selectedTables() {
     return $this->_selectedTables;
   }
 
+  /**
+   * Add campaign fields.
+   *
+   * @param bool $groupBy
+   *   Add GroupBy? Not appropriate for detail report.
+   * @param bool $orderBy
+   *   Add GroupBy? Not appropriate for detail report.
```

Comment:
@yashodha minor nitpick here the doc should probably say orderBy here not groupBy

Reply:
@seamuslee001 good catch :)